airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```
@@ -1789,7 +1789,7 @@
 - name: Stripe
   sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   dockerRepository: airbyte/source-stripe
-  dockerImageTag: 3.0.0
+  dockerImageTag: 3.0.1
```

```suggestion
  dockerImageTag: 3.0.0
```

This will get bumped automatically.